com.github.sixpounder.GameOfLife.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id" : "com.github.sixpounder.GameOfLife",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "game-of-life",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```

Comment:
build need to be offline. this is not allow (and will fail)

Reply:
Forgot to remove it, sorry. Now it's done.